cmd/otelcol/config/collector/agent_config.yaml

```
@@ -156,7 +156,9 @@ exporters:
     endpoint: "${SPLUNK_GATEWAY_URL}:4317"
     tls:
       insecure: true
-  # Debug
+  # Debug exporter
+  # If using version 0.85 or lower of the Collector
+  # replace 'debug' with 'logging'
   debug:
```

Since this is a bundled, default config file I suggest we postpone adopting the new name.

```suggestion
  # This component has been deprecated by the "debug" exporter,
  # which will be adopted in a future release.
  logging:
```